src/runtime/runtime.ts

```diff
@@ -288,86 +336,357 @@ class Scan {
    * (i.e., this scan could not be satisfied due to proposals from previous scans) we'll return false.
    */
   applyInput(input:Change, prefix:ID[]) {
-    let resolved = this.resolve(prefix);
-
     // If this change isn't relevant to this scan, skip it.
-    if(this.notStaticMatch(input, "e")) return true;
-    if(this.notStaticMatch(input, "a")) return true;
-    if(this.notStaticMatch(input, "v")) return true;
-    if(this.notStaticMatch(input, "n")) return true;
+    if(this.notStaticMatch(input, "e")) return ApplyInputState.none;
```

Comment:
The function comment still documents returned values as true or false. Needs to be updated.